packages/kit/src/exports/vite/index.js

```diff
@@ -775,6 +783,24 @@ async function kit({ svelte_config }) {
 						input[name] = path.resolve(file);
 					});
 
+					// ...and the server tracing file
+					// TODO: Resolve this however we resolve all of the other files after we deprecate the `files` options
+					const server_tracing = resolve_entry(path.resolve(cwd, 'src/tracing.server'));
```

Comment:
TODO we should wait until https://github.com/sveltejs/kit/pull/14152 is merged

```suggestion
					const server_tracing = resolve_entry(path.join(kit.files.src, 'tracing.server'));
```